specification/regionmove/resource-manager/Microsoft.Migrate/preview/2019-10-01-preview/regionmovecollection.json

```diff
@@ -0,0 +1,1929 @@
+    },
+    "NetworkInterfaceResourceSettings": {
+      "description": "Defines the network interface resource settings.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ResourceSettings"
+        }
+      ],
+      "properties": {},
+      "x-ms-discriminator-value": "Microsoft.Network/networkInterfaces"
+    },
+    "SqlServerResourceSettings": {
+      "description": "Defines the SQL Server resource settings.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ResourceSettings"
+        }
+      ],
+      "properties": {},
+      "x-ms-discriminator-value": "Microsoft.Sql/servers"
+    },
+    "ZoneRedundant": {
+      "description": "Defines the zone redundant resource setting.",
+      "enum": [
+        "Enable",
+        "Disable"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "ZoneRedundant",
+        "modelAsString": true
+      }
+    },
+    "SqlElasticPoolResourceSettings": {
+      "description": "Defines the Sql ElasticPool resource settings.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ResourceSettings"
+        }
+      ],
+      "properties": {
+        "zoneRedundant": {
+          "$ref": "#/definitions/ZoneRedundant"
+        }
+      },
+      "x-ms-discriminator-value": "Microsoft.Sql/servers/elasticPools"
+    },
+    "SqlDatabaseResourceSettings": {
+      "description": "Defines the Sql Database resource settings.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ResourceSettings"
+        }
+      ],
+      "properties": {
+        "zoneRedundant": {
+          "$ref": "#/definitions/ZoneRedundant"
+        }
+      },
+      "x-ms-discriminator-value": "Microsoft.Sql/servers/databases"
+    },
+    "ResourceGroupResourceSettings": {
+      "description": "Defines the resource group resource settings.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ResourceSettings"
+        }
+      ],
+      "properties": {},
+      "x-ms-discriminator-value": "resourceGroups"
+    },
+    "PublicIPAddressResourceSettings": {
+      "description": "Defines the public IP address resource settings.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ResourceSettings"
+        }
+      ],
+      "properties": {},
+      "x-ms-discriminator-value": "Microsoft.Network/publicIPAddresses"
+    },
+    "LoadBalancerResourceSettings": {
+      "description": "Defines the load balancer resource settings.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ResourceSettings"
+        }
+      ],
+      "properties": {},
+      "x-ms-discriminator-value": "Microsoft.Network/loadBalancers"
+    },
+    "CloudError": {
+      "x-ms-external": true,
+      "properties": {
+        "error": {
```

Comment:
@pilor  Currently, the service returns cloud error with the "error" property named as "properties".
However the ARM/TemplateEngine does not crib, on errors when we send this as "properties" field instead of the "error" field?



`    /// <summary>
    /// Defines the object representing an error instance.
    /// </summary>
    public class CloudError
    {
        /// <summary>
        /// Initializes a new instance of the <see cref="CloudError"/> class.
        /// </summary>
        public CloudError()
        {
        }

        /// <summary>
        /// Initializes a new instance of the <see cref="CloudError"/> class.
        /// </summary>
        /// <param name="error">The error.</param>
        /// <param name="culture">The specific culture in which the message is needed.</param>
        public CloudError(CloudErrorProperty error, string culture)
        {
            this.Properties = new CloudErrorBody(error, culture);
        }

        /// <summary>
        /// Gets or sets the cloud error properties.
        /// </summary>
        [JsonProperty(PropertyName = "properties")]
        public CloudErrorBody Properties { get; set; }
    }`